en/docs/integrations/build-your-first-integration.md

```diff
@@ -71,7 +70,7 @@ Follow this procedure to connect to the world bank API and retrieve population d
         |----------------|-------------------------------|
         | **Type**       | `int`                         |
         | **Name**       | `population`                  |
-        | **Expression** | `(populationByCountry[0]?.value ?: 0) / 1000000`|
+        | **Expression** | `populationByCountry[0]?.value ?: 0 /1000000`|
```

Comment:
We need to apply this change to the code block too right?

Reply:
Fixed. Thanks